FingerBlox/app/src/main/java/org/fingerblox/fingerblox/CameraView.java

```diff
@@ -98,4 +101,13 @@ protected void fixFocusToggle() {
         }
         mCamera.setParameters(parameters);
     }
+
+    public Camera.Parameters getCameraParameters() {
+        try {
+            return mCamera.getParameters();
+        } catch (NullPointerException e) {
+            Log.i(TAG, "Could not retrieve camera parameters. Camera in CameraView == NULL");
```

Comment:
I had it return null now to use with the try/catch, as both mCamera and getParameters() can be null.